.gitignore

```diff
@@ -85,7 +85,9 @@ Session.vim
 *~
 
 # VSCode project files
-.vscode
+!.vscode/
```

Comment:
Do we need this line?

Reply:
This will start checking in everything in `.vscode` including editor preferences. We don't want that.